### ISSUE TITLE
uninitialized constant ReadmooDL::API::Http (NameError)

### ISSUE THREAD

**Issue by NomadThanatos** [msglight4874/readmoo-dl#4]
I have tried this on both Windows on Linux, this only occurs on Windows.
I think it might be caused by dependency issue. 

I modify Http to HTTP to solve this issue in https://github.com/msglight4874/readmoo-dl/blob/master/lib/readmoo_dl/API.rb#L10

I have checked the other part in API.rb, I think this should be changed also for consistency.

```
PS C:\Users\Nomad\Program\readmoo-dl> bundle exec ruby main.rb
正在取得檔案路徑...Traceback (most recent call last):
        5: from main.rb:8:in `<main>'
        4: from C:/Users/Nomad/Program/readmoo-dl/lib/readmoo_dl/downloader.rb:23:in `download'
        3: from C:/Users/Nomad/Program/readmoo-dl/lib/readmoo_dl/downloader.rb:37:in `job'
        2: from C:/Users/Nomad/Program/readmoo-dl/lib/readmoo_dl/downloader.rb:23:in `block in download'
        1: from C:/Users/Nomad/Program/readmoo-dl/lib/readmoo_dl/downloader.rb:42:in `fetch_base_file'
C:/Users/Nomad/Program/readmoo-dl/lib/readmoo_dl/API.rb:10:in `fetch': uninitialized constant ReadmooDL::API::Http (NameError)
```

**Comment by msglight4874** [comment]
HI @NomadThanatos ,

I don't have windows pc, so thank you for figuring out and fix this issue.

By the way, I made chrome extension version recently. Maybe your can try it.

https://github.com/msglight4874/epub-exporter

**Comment by NomadThanatos** [comment]
I have tried that chrome extension, pretty nice, but sometimes it will fail to download. (readmoo-dl is the one I have never met any download failure so far.

The main feature I want is to maintain a download history automatically, I got it to work, but I am not that familiar with Ruby, so it is somehow ugly.

If you do not think it is such ugly, I can make another pull request for this feature.

https://github.com/NomadThanatos/readmoo-dl/commit/93c1d4781f7d94c0e9520cbfb0974264c280cfc3
